tests/suites/test_suite_bignum_core.misc.data

```diff
@@ -491,6 +491,834 @@ mpi_core_fill_random:42:0:-1:0:MBEDTLS_ERR_MPI_BAD_INPUT_DATA
 Fill random core: 42 bytes, 5 missing limbs
 mpi_core_fill_random:42:0:-5:0:MBEDTLS_ERR_MPI_BAD_INPUT_DATA
 
+mbedtls_mpi_core_mul #1: 0x0 * 0x0 = 0x0
+mpi_core_mul:"0":"0":"0"
```

Comment:
Should these test cases be generated by the Python script now? Or should we raise an issue to do that "later"?

Reply:
Yes, it should. It was only a rebase.